components/centraldashboard/public/components/logout-button.js

```diff
@@ -10,72 +9,26 @@ import '@polymer/paper-button/paper-button.js';
  * AAW: Not using this compoent and the logoutURL env param. Doesn't work.
  * https://github.com/StatCan/kubeflow/issues/152
  */
-
 export class LogoutButton extends PolymerElement {
     static get template() {
-        return html`
-            <paper-button id="logout-button" on-tap="logout">
-                <iron-icon icon='kubeflow:logout' title="Logout">
-                </iron-icon>
-            </paper-button>
-            <iron-ajax
-                id='logout'
-                url$='{{logoutUrl}}'
-                method='post'
-                handle-as='json'
-                headers='{{headers}}'
-                on-response='_postLogout'>
-            </iron-ajax>
-        `;
-    }
-
-    static get properties() {
```

Comment:
I see a lot of stuff getting deleted here for the logout button. Will it still work? Like, there is still the "logoutUrl" defined in the html, but the properties are getting deleted. So, will the logoutUrl still be defined for the html?  I also see some headers stuff that seem to work with our auth service. That might still be needed. (Although, I think I remember you said that the auth service might be changing with this version update?)

Reply:
Truthfully, I am unsure if it will work or behave, but they did all those changes upstream in this commit https://github.com/kubeflow/kubeflow/pull/7654/files. 

I figured we will need to test once in dev and confirm